templates/phpcs.xml.dist

```diff
@@ -2,6 +2,9 @@
 <ruleset name="WordPress Coding Standards for Plugins">
 	<description>Generally-applicable sniffs for WordPress plugins</description>
 
+	<rule ref="PHPCompatibility"/>
```

Comment:
Instead of pulling in the general PHPCodeSniffer rule, you should pull in the Framework-specific version for WordPress, which contains some exclusions to avoid false positives. This is needed because WordPress includes a lot of compatibility code and polyfills, so the checks need to take this into account. You can see the current exclusions here: https://github.com/wimg/PHPCompatibility/blob/master/framework-rulesets/wordpress.xml

Here's the adapted rule to use:
```
<rule ref="./vendor/wimg/php-compatibility/framework-rulesets/wordpress.xml"/>
```

Reply:
Thank you for your reviewing.

Can `vendor` directory place at `template` directory?

Reply:
No, that would be pulled in via Composer. But now that you're mentioning it, I don't think we're using Composer for the scaffolded plugins...

Let's merge this PR as-is for now and create an issue for evaluating how to use the Framework-specific ruleset in the future.

Reply:
Can I see other plugin which uses Composer, and adapt it for scaffolded?

Reply:
I don't think adding Composer support directly is a good idea at this point. Most plugin developers will not know what this means.

This needs more thought and exploration first. I've created a corresponding issue for now: https://github.com/wp-cli/scaffold-command/issues/155